backend/v2/allocations/repositories.py

```diff
@@ -169,7 +169,7 @@ async def get_all_allocations_for_epoch(
     )
 
     if not include_zero_allocations:
-        query = query.filter(Allocation.amount > 0)
+        query = query.filter(Allocation.amount != "0")
```

Comment:
`query = query.filter(int(Allocation.amount) > 0)`

Reply:
Yes, but we have bigints and casting happens on db level - so not sure if it would get truncated down to et int32. String cmp is all we currently need so will revisit this thing later on